lib/std/json.zig

```diff
@@ -1483,7 +1483,22 @@ fn parseInternal(comptime T: type, token: Token, tokens: *TokenStream, options:
                                         parseFree(field.field_type, @field(r, field.name), options);
                                     }
                                 }
-                                @field(r, field.name) = try parse(field.field_type, tokens, options);
+                                var parsed_value = try parse(field.field_type, tokens, options);
+                                if (!field.is_comptime) {
+                                  @field(r, field.name) = parsed_value;
```

Comment:
I wonder if this ends up with an extra copy due to lack of result-location semantics

Reply:
I do not know what that means :D but I am [reading this](https://github.com/ziglang/zig/issues/3794) and attempting to understand.

Reply:
Is the solution (if it is a problem) to just call `try parse` twice, once in each branch?